module_presentation/src/main/java/project/redis/movie/controller/MovieController.java

```diff
@@ -1,24 +1,33 @@
 package project.redis.movie.controller;
 
+import jakarta.validation.constraints.Size;
 import java.util.List;
 import lombok.RequiredArgsConstructor;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
 import project.redis.movie.dto.NowPlayMovieDto;
-import project.redis.movie.service.MovieService;
+import project.redis.movie.service.MovieQueryService;
 
 @RestController
 @RequestMapping("/api/v1/movie")
 @RequiredArgsConstructor
 public class MovieController {
 
-    private final MovieService movieService;
+    private final MovieQueryService movieQueryService;
+    //private final MovieService movieService;
 
     @GetMapping("/now-playing")
-    public ResponseEntity getNowPlayingMovies() {
-        List<NowPlayMovieDto> nowPlayMovieDtos = movieService.getNowPlayingMovies();
+    public ResponseEntity<List<NowPlayMovieDto>> getNowPlayingMovies(
+            @RequestParam(required = false, name = "movieTitle")
```

Comment:
현재 getMapping에서  /now-playing 처럼 kebab-case를 사용하고 계신데요
requestparam도 통일해 주시는 것이 좋습니다! 
아래와 같읕 방식으로 수정해 보세요! 
```suggestion
            @RequestParam(required = false, name = "movie-title")
```